packages/next/src/utils.ts

```diff
@@ -2213,6 +2213,7 @@ export const onPrerenderRoute =
           fallback: jsonFsRef,
           group: prerenderGroup,
           bypassToken: prerenderManifest.bypassToken,
+          experimentalBypassFor,
```

Comment:
Looks like we also need to parse this value for `fallbackRoutes` so it's available once we have it in the `dynamicRoutes` field as well

x-ref: https://github.com/vercel/vercel/blob/1abda9ca871ce4fad98b260c9467fd5a7f294a40/packages/next/src/utils.ts#L1134-L1152